### PR TITLE
Issue 27 add modeltemplate component

### PR DESCRIPTION
**Pull request by muditchoudhary**
Close: #27 

Changes made:

1. Added ModalTemplate.tsx fle
2. Import & add the ModalTemplate in Layout tag 
3. Add ModalTemplate to the config file.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/65"></a>

### CODE REVIEW

File: components/ContentTemplates/ModalsTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react"
+
+const ModalsTemplate = () => {
+	return <div>MenusTemplate</div>
```
Comment (EmmaDawsonDev) [inline components/ContentTemplates/ModalsTemplate.tsx:4]:
```suggestion
	return <div>ModalsTemplate</div>
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Just a small copy-paste mistake to fix in the component then this is ready to merge :)

**Comment by muditchoudhary** [comment]
@EmmaDawsonDev fixed!! see it's good :)

**Review by EmmaDawsonDev — APPROVED** [review]
Looks great, thank you!

**Pull request merged**